ui/src/app/admin/controllers/AdminCtrl.js

```diff
@@ -7,6 +7,36 @@
   function AdminCtrl($scope, ToastService, AdminService) {
 
     $scope.signup_requests = [];
+    $scope.usageChartData = [];
+
+    var formatBytes = function(bytes, decimals) {
+       if(bytes == 0) return '0 Byte';
+       var k = 1000;
```

Comment:
use following format. Also use a meaningful word instead of 'i'

```javascript
var k=1000,
    dm = decimals +1 ||3,
    sizes = [..... ],
    i = .... ;
```